README.md

```diff
@@ -96,6 +96,37 @@ You can also set the following mount points by passing the `-v /host:/container`
 directory has the appropriate permissions and that the owner and group of the directory
 matches the user UID or name which is running inside the container.**
 
+Inclusion of the `oc` binary
+---------------------------------
+
+To assist in interacting with the OpenShift API server while using this image, the `oc` binary, the CLI command for OpenShift, has been
+installed in this image.
+
+However, it needs to noted that backward compatibility is not guaranteed between different versions of `oc` and the OpenShift
+API Server.  As such, it is recommended that you align versions of this image present in the nodes of your cluster with your 
+OpenShift API server.  In other words, you should use the version specific tag instead of the `latest` tag.
+
+|  Jenkins image version      | `oc` client version      |
+| :-------------------------- | ------------------------ |
+|  `jenkins-*-centos7:v3.7`   | 3.7 `oc` binary          |
+|  `jenkins-*-centos7:v3.6`   | 3.6 `oc` binary          |
+|  `jenkins-*-centos7:latest` | 3.9 `oc` binary          |
+|  `jenkins-*-rhel7:v3.7`     | 3.7 `oc` binary          |
+|  `jenkins-*-rhel7:v3.6`     | 3.6 `oc` binary          |
+|  `jenkins-*-rhel7:latest`   | 3.6 `oc` binary          |
+
+
+**Notice:  There is an additional consideration with the pod configurations for the Kubernetes Plugin; earlier versions of this image
+did not specify the "pull always" policy for the default agents/slaves configured.  As a result, users may have older/different images on 
+your nodes depending when the images were pulled.  Staring with the 3.7 release, the default changed to "pull always" to avoid this problem 
+in the future.  But if you started using this image prior to 3.7, verification of your Kubernetes plugin configurations for the image pull 
+policy used is warranted to guarantee consistency around what image is being used on each of your nodes.
+
+**Notice: the `latest` tag for the RHEL7 images will point to 3.6 for the time being in order to support users whose pull policies for
+their Kubernetes Plugin configuration lack the "pull always" setting.  This way, they will have an older `oc` client which should be able to 
```

Comment:
it's unrelated to pullalways.  The rhel7 images point to the 3.6 tag (not just for the time being but probably forever) to ensure users on older clusters and who have older slave configurations that point to ":latest" will continue to work (will continue to pick up the v3.6 oc binary).  

New slave configurations use version specific tags (as well as pullalways) to ensure the right image is used for their cluster.

Reply:
I was trying to be as non-committal as possible wrt "time being" vs. "probably forever"; just seemed rash to say "forever"... perhaps just remove the "time being" phrase and not substitute any other indication of time frame ?

otherwise sure I'll remove the pull always piece, just reference "to ensure users on older clusters and who have older slave configurations that point to ":latest" will continue to work (will continue to pick up the v3.6 oc binary)."

Reply:
Maybe "indefinitely" or "for the foreseeable future" instead.  "For the time being" implies to me that we have plans to change it at some point when we get to it, which we don't.

Reply:
I'll go with indefinitely